Resources/Prototypes/Entities/Mobs/NPCs/spacetick.yml

```diff
@@ -60,22 +60,20 @@
       autoPopulate: false
       name: action-name-disarm
   - type: MeleeWeapon
+    range: 1.3
     hidden: true
     soundHit:
         path: /Audio/Effects/bite.ogg
     angle: 0
     animation: WeaponArcBite
     damage:
-      groups:
-        Brute: 2
-  - type: SolutionContainerManager
-    solutions:
-      melee:
-        reagents:
-        - ReagentId: Toxin
-          Quantity: 5
-  - type: MeleeChemicalInjector
-    solution: melee
+      types:
+        Piercing: 2
+        Blunt: 1
+  - type: MeleeBloodletter
+    bloodReduction:
+      types:
+        Slash: 10.0
```

Comment:
doesn't this make ticks a lot more generic than when they used to inject toxin...?